sample/src/sample/service.clj

```diff
@@ -219,10 +217,11 @@
         {:post add-order}
         ["/:id" ^:interceptors [load-order-from-db]
          {:get get-order-by-id}]]
+
        ["/secure" ^:interceptors [basic-auth] {:delete delete-db}]
 
        ["/doc" {:get [(swagger/swagger-doc)]}]
-       ["/*resource" {:get [(swagger/swagger-ui)]}]]]]))
+       ["/swagger-ui/*resource" {:get [(swagger/swagger-ui)]}]]]]))
```

Comment:
So it's not possible to use splat params at the root of the project? That sucks, I liked the swagger ui mounted on "/"